reference/commands.rst

```diff
@@ -17,12 +17,3 @@ Creator commands
 ----------------
 
 
-
-Custom commands
```

Comment:
I think that we should at least mention the custom commands from here and provide a link to the new location

Reply:
Definitely

Reply:
Done